### PR TITLE
Update Readme.txt tested up to 6.4

### PR DESCRIPTION
**Pull request by bph**
## What?
Update the version in "Tested up to" section of the readme.txt of the plugin to the latest WordPress version.

## Why?
To avoid the _"This plugin hasn't been tested for your current version of WordPress"_ banner in the plugin repo and plugin workflow on WPAdmin

### CODE REVIEW

File: packages/create-block/README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ For example, running the `start` script from inside the generated folder (`npm s
 
 ## External Project Templates
 
-[Click here](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block-tutorial-template/) for information on External Project Templates
+[Click here](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/packages-create-block-external-template/) for information on External Project Templates
```
Comment (gziolo) [inline packages/create-block/README.md:144]: It doesn't look relevant based on the PR's description, but it's a good change we can include.

**Review by gziolo — APPROVED** [review]

File: packages/create-block/README.md (hunk 1 above)
Comment (bph) [reply packages/create-block/README.md:144]: A I seem to have goofed this simple thing... how do I remove that file?

Comment (bph) [reply packages/create-block/README.md:144]: Curious. This change was already merged in #56106, but it's not harmful to do it again

**Comment by youknowriad** [comment]
Just noting that this PR had an impact on the performance job, it now started failing on trunk. In general when we update the "Tested up to", I think we should probably change the "base" commit used for performance jobs.

**Comment by gziolo** [comment]
It looks like after https://github.com/WordPress/gutenberg/commit/352ab9993ed4eefd0d11f15dc4146283ec2080ff, there are the same fatal errors reported. Can you explain how a change in the readme file impacts the performance job so we could document it?

**Comment by youknowriad** [comment]
The "Tested up to" flag is used as a "base WordPress version" for the docker containers that are used in the performance jobs. The reason is that the performance jobs compare current commit with an old commit and upon release, the compare current release with WP version - 1. So the WP version need to be one that is older than WP version - 1 but it also need to be one that is supported by the Gutenberg plugin today.

**Comment by youknowriad** [comment]
That said, I'm not out of ideas about the current failure, we need to debug further.

**Comment by bph** [comment]
That's super interesting to learn. Good luck troubleshooting this. 🤗

**Pull request merged**